docs/api-docs/cart-and-checkout/working-sf-apis.md

```diff
@@ -3,1078 +3,1023 @@
-	}],
-	"lineItems": {
-		"physicalItems": [{
-			"id": "7349b13a-1453-4050-a769-1a6ad1823369",
-			"parentId": null,
-			"variantId": 362,
-			"productId": 191,
-			"sku": "",
-			"name": "Openhouse No. 3",
-			"url": "https://{store_url)/all/openhouse-no-3/",
-			"quantity": 1,
-			"brand": "Openhouse Magazine",
-			"isTaxable": true,
-			"imageUrl": "https://cdn11.bigcommerce.com/s-{store_hash)/products/191/images/475/openhousevol3_1024x1024__59692__16355.1534344544.330.500.jpg?c=2",
-			"discounts": [],
-			"discountAmount": 0,
-			"couponAmount": 0,
-			"listPrice": 27.95,
-			"salePrice": 27.95,
-			"extendedListPrice": 27.95,
-			"extendedSalePrice": 27.95,
-			"isShippingRequired": true,
-			"type": "physical",
-			"giftWrapping": null
-		}, {
-			"id": "4a69cbdf-4320-4e1f-852b-0edc2a55f13a",
-			"parentId": null,
-			"variantId": 356,
-			"productId": 185,
-			"sku": "",
-			"name": "Utility Caddy",
-			"url": "https://{store_url)/all/utility-caddy/",
-			"quantity": 1,
-			"brand": "OFS",
-			"isTaxable": true,
-			"imageUrl": "https://cdn11.bigcommerce.com/s-{store_hash)/products/185/images/449/utilitybucket1_1024x1024__78563__75042.1534344535.330.500.jpg?c=2",
-			"discounts": [],
-			"discountAmount": 0,
-			"couponAmount": 0,
-			"listPrice": 46,
-			"salePrice": 46,
-			"extendedListPrice": 46,
-			"extendedSalePrice": 46,
-			"isShippingRequired": true,
-			"type": "physical",
-			"giftWrapping": null
-		}],
-		"digitalItems": [],
-		"giftCertificates": [],
-		"customItems": []
-	},
-	"createdTime": "2018-11-06T19:22:51+00:00",
-	"updatedTime": "2018-11-06T19:22:51+00:00"
+ "id": "d4e978c2-bdcf-41b0-a49b-fecf4f5223c1",
+ "customerId": 0,
+ "email": "",
+ "currency": {
+   "name": "US Dollars",
+   "code": "USD",
+   "symbol": "$",
+   "decimalPlaces": 2
+ },
+ "isTaxIncluded": false,
+ "baseAmount": 274.5,
+ "discountAmount": 0,
+ "cartAmount": 274.5,
+ "coupons": [],
+ "discounts": [
+	...
+ ],
+ "lineItems": {
+   "physicalItems": [
+     {
+       "id": "57a877e0-d898-47d0-910d-88656e8dee0c",
+       "parentId": null,
+       "variantId": 66,
+       "productId": 86,
+       "sku": "ABS",
+       "name": "[Sample] Able Brewing System",
+       "url": "https://{store_url}/all/able-brewing-system/",
+       "quantity": 1,
+		...
+       "extendedSalePrice": 225,
+       "isShippingRequired": true,
+       "type": "physical",
+       "giftWrapping": null
+     },
+     {
+       "id": "22c461a2-eff9-4b72-8d22-7c2792ce2c2d",
+       "parentId": null,
+       "variantId": 67,
+       "productId": 88,
+       "sku": "CC3C",
+       "name": "[Sample] Chemex Coffeemaker 3 Cup",
+       "url": "https://{store_url}/all/chemex-coffeemaker-3-cup/",
+       "quantity": 1,
+		...
+       "extendedSalePrice": 49.5,
+       "isShippingRequired": true,
+       "type": "physical",
+       "giftWrapping": null
+     }
+   ],
+  ...
+ },
+ ...
 }
 ```
+Please take note of the value of the `cartId` as it will be used later in the tutorial. 

```

Comment:
```suggestion
<div class="HubBlock--callout">
<div class="CalloutBlock--info">
<div class="HubBlock-content">

>Note: Please take note of the value of the `cartId` as it will be used later in the tutorial.

</div>
</div>
</div>
```